### PR TITLE
OKTA-262734: use access token guide: add a note on token renewal

### PR DESCRIPTION
**Pull request by oleksandrpravosudko-okta**
## Description:
- **What's changed?** 
Use the access token [guide](https://developer.okta.com/docs/guides/sign-into-spa/angular/use-the-access-token/) is updated with token renewal hints.
- **Is this PR related to a Monolith release?** 
No

### Resolves:

* [OKTA-262734](https://oktainc.atlassian.net/browse/OKTA-262734)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md [hunk 1]
```diff
@@ -14,4 +14,11 @@ On your back-end (the API), make sure that you check for valid tokens in incomin
 
 <StackSelector snippet="getaccesstoken"/>
 
+<br/>
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md:17]: This shouldn't be necessary.

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md [hunk 2]
```diff
@@ -14,4 +14,11 @@ On your back-end (the API), make sure that you check for valid tokens in incomin
...
+
+In order to enable access token renewal it is required to obtain a refersh token. See [Get a refresh token with the code flow](/docs/guides/refresh-tokens/get-refresh-token/#get-a-refresh-token-with-the-code-flow).
```
Comment (JayNewstrom) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md:19]:
```suggestion
In order to enable access token renewal it is required to obtain a refresh token. See [Get a refresh token with the code flow](/docs/guides/refresh-tokens/get-refresh-token/#get-a-refresh-token-with-the-code-flow).
```

Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md:19]:
```suggestion
In order to enable access token renewal you must obtain a refresh token. See [Get a refresh token with the code flow](/docs/guides/refresh-tokens/get-refresh-token/#get-a-refresh-token-with-the-code-flow).
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md [hunk 3]
```diff
@@ -14,4 +14,11 @@ On your back-end (the API), make sure that you check for valid tokens in incomin
...
+Note that using refresh token with SPA is an Early Access feature. To enable it, contact [Support](https://support.okta.com/help/open_case).
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md:20]:
```suggestion
Note that using refresh token with a SPA is an Early Access feature. To enable it, contact [Support](https://support.okta.com/help/open_case).
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md [hunk 4]
```diff
@@ -14,4 +14,11 @@ On your back-end (the API), make sure that you check for valid tokens in incomin
...
+
+Alternatively, tokens can be renewed by hitting `/authorize` endpoint. See [Get a new access token/ID token silently for your SPA ](/docs/guides/refresh-tokens/get-refresh-token/#get-a-new-access-token-id-token-silently-for-your-spa)
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md:22]:
```suggestion
Alternatively, tokens can be renewed by hitting the `/authorize` endpoint. See [Get a new access token/ID token silently for your SPA ](/docs/guides/refresh-tokens/get-refresh-token/#get-a-new-access-token-id-token-silently-for-your-spa)
```

**Review by jakubvul-okta — CHANGES_REQUESTED** [review]

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md (hunk 2 above)
Comment (oleksandrpravosudko-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md]: thanks - this got included into one of the suggested changes below

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md [hunk 5]
```diff
@@ -12,6 +12,11 @@ Authorization: Bearer {token}
 ```
 On your back-end (the API), make sure that you check for valid tokens in incoming requests. See [Protect your API endpoints](/docs/guides/protect-your-api/).
 
-<StackSelector snippet="getaccesstoken"/>
+<StackSelector snippet="getaccesstoken"/> 
+
+In order to enable access token renewal you must obtain a refresh token. See [Get a refresh token with the code flow](/docs/guides/refresh-tokens/get-refresh-token/#get-a-refresh-token-with-the-code-flow).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md:17]:
```suggestion
To enable access token renewal you must obtain a refresh token. See [Get a refresh token with the code flow](/docs/guides/refresh-tokens/get-refresh-token/#get-a-refresh-token-with-the-code-flow).
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md [hunk 6]
```diff
@@ -12,6 +12,11 @@ Authorization: Bearer {token}
...
+Note that using refresh token with a SPA is an Early Access feature. To enable it, contact [Support](https://support.okta.com/help/open_case).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md:18]:
```suggestion
> **Note:** Using a refresh token with a SPA is an Early Access feature. To enable it, contact [Support](https://support.okta.com/help/open_case).
```

**Review by susanharper-okta — COMMENTED** [review]
couple edits

**Review by susanharper-okta — APPROVED** [review]
LGTM!

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md [hunk 7]
```diff
@@ -12,6 +12,11 @@ Authorization: Bearer {token}
 ```
 On your back-end (the API), make sure that you check for valid tokens in incoming requests. See [Protect your API endpoints](/docs/guides/protect-your-api/).
 
-<StackSelector snippet="getaccesstoken"/>
+<StackSelector snippet="getaccesstoken"/> 
+
+To enable access token renewal you must obtain a refresh token. See [Get a refresh token with the code flow](/docs/guides/refresh-tokens/get-refresh-token/#get-a-refresh-token-with-the-code-flow).
+> **Note:** Using a refresh token with a SPA is an Early Access feature. To enable it, contact [Support](https://support.okta.com/help/open_case).
+
+Alternatively, tokens can be renewed by hitting the `/authorize` endpoint. See [Get a new access token/ID token silently for your SPA ](/docs/guides/refresh-tokens/get-refresh-token/#get-a-new-access-token-id-token-silently-for-your-spa)
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/use-the-access-token/index.md:20]:
```suggestion
Alternatively, tokens can be renewed by hitting the `/authorize` endpoint. See [Get a new access token/ID token silently for your SPA ](/docs/guides/refresh-tokens/get-refresh-token/#get-a-new-access-token-id-token-silently-for-your-spa).
```

**Review by jakubvul-okta — APPROVED** [review]

**Pull request merged**